src/main/java/com/socialguide/hive/udf/DateForTimeZone.java

```diff
@@ -9,7 +9,7 @@
 import java.util.TimeZone;
 import java.text.ParseException;
 
-class DateForTimeZone extends UDF {
+public final class DateForTimeZone extends UDF {
```

Comment:
Is there any particular reason for making these classes `final`?

Reply:
Hive requires UDF classes to be `final`. https://cwiki.apache.org/confluence/display/Hive/HivePlugins